net-p2p/syncthing/syncthing-0.14.11.ebuild

```diff
@@ -67,8 +66,9 @@ src_install() {
 	if use tools ; then
 		dobin bin/syncthing
 		exeinto /usr/libexec/syncthing
+		local exe
 		for exe in bin/* ; do
-			[[ "${exe}" = "bin/syncthing" ]] || doexe "${exe}"
+			[[ "${exe}" == "bin/syncthing" ]] || doexe "${exe}"
 		done
 	else
 		dobin syncthing
```

Comment:
can you simplify the following commands? i.e.
```
systemd_dounit etc/linux-systemd/system/${PN}@.service \
 		etc/linux-systemd/system/${PN}-resume.service
```
or even
```
systemd_dounit etc/linux-systemd/system/${PN}{@,-resume}.service
```
if you're feeling adventurous